mne/viz/_brain/_timeviewer.py

```diff
@@ -473,7 +476,7 @@ def play(self):
             try:
                 self._advance()
             except Exception:
-                self.playback = False
```

Comment:
I wonder if we can be a little more specific regarding which exception to catch?

Reply:
Actually the goal here is to stay as general as possible to behave like a fail-safe, catch any unexpected Qt failures without crashing the Python interpreter. You can take a look at `safe_event()`:

https://github.com/mne-tools/mne-python/blob/2fc2e704976e7319ea200c1a979ad9392fd297c5/mne/viz/_brain/_timeviewer.py#L25-L31

Reply:
Haha alright then :)